superduper/base/config.py

```diff
@@ -270,6 +270,43 @@ class Downloads(BaseConfig):
     timeout: t.Optional[int] = None
 
 
+@dc.dataclass
+class LogConfig(BaseConfig):
+    """Describes the configuration for logging.
+
+    :param default_format: The default log format
+    :param ray_format: The log format for Ray deployments
+    :param use_colors: Whether to use colors in the log output
+    """
+
+    default_format: str = (
```

Comment:
I wouldn't do it like this. 
I would add a variable to the config `colorize` and then in the logging module do one or the other.

Reply:
**`config.py`**

```python
colorize: bool = True
```

**`logging.py`**

```python
if CFG.logging.colorize:
    message = (
        "<green>{time:YYYY-MMM-DD HH:mm:ss.SS}</green>"
        "| <level>{level: <8}</level> "
        "| <cyan>{extra[hostname]: <8}</cyan>"
        "| <cyan>{name}</cyan>:<cyan>{line: <4}</cyan> "
        f"| <level>{message}</level>"
    )
else:
    message = (
        "{time:YYYY-MMM-DD HH:mm:ss.SS}"
        "| {level: <8} "
        "| {name}:{line: <4} "
        f"| {message}"
    )
```

Reply:
> **`config.py`**
> 
> ```python
> colorize: bool = True
> ```
> 
> **`logging.py`**
> 
> ```python
> if CFG.logging.colorize:
>     message = (
>         "<green>{time:YYYY-MMM-DD HH:mm:ss.SS}</green>"
>         "| <level>{level: <8}</level> "
>         "| <cyan>{extra[hostname]: <8}</cyan>"
>         "| <cyan>{name}</cyan>:<cyan>{line: <4}</cyan> "
>         f"| <level>{message}</level>"
>     )
> else:
>     message = (
>         "{time:YYYY-MMM-DD HH:mm:ss.SS}"
>         "| {level: <8} "
>         "| {name}:{line: <4} "
>         f"| {message}"
>     )
> ```

Ok,I have changed